.github/workflows/build-status.yml

```diff
@@ -339,6 +339,7 @@ jobs:
               echo "ERROR: HEAD is not the expected commit hash"
               exit 1
           fi
+          mkdir -p "$NVS_HOME/cache" && touch "$NVS_HOME/cache/node" #Faking a node file to avoid bootstraping on first use https://github.com/jasongin/nvs/blob/d1df6ef535df2600177732c5cc446259ffdb94a4/nvs.sh#L77
```

Comment:
```suggestion
          mkdir -p "$NVS_HOME/cache" && ln -s "$(which node)" "$NVS_HOME/cache/node" #Faking a node file to avoid bootstraping on first use https://github.com/jasongin/nvs/blob/d1df6ef535df2600177732c5cc446259ffdb94a4/nvs.sh#L77
```